.github/workflows/check-action-typing.yml

```diff
@@ -0,0 +1,16 @@
+name: Check Action Typing
+
+on:
+  - push
+  - pull_request
```

Comment:
we enforce changes to go through pull requests, so `pull_request` should suffice